app/main.py

```diff
@@ -1,3 +1,70 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
+            raise ValueError("Unsupported operand type")
+
+    def __iadd__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise ValueError("Unsupported operand type")
+        return self
+
+    def __mul__(self, scalar: int | float | None) -> None:
+        if isinstance(scalar, (int, float)):
+            return Distance(self.km * scalar)
+        else:
+            raise TypeError("Unsupported operand type")
+
+    def __truediv__(self, divisor: int) -> None:
+        if isinstance(divisor, (int, float)):
+            result = self.km / divisor
+            return Distance(round(result, 2))
+        else:
+            raise TypeError("Unsupported operand type")
+
+    def __lt__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        else:
+            return self.km < other
```

Comment:
`else` block is redundant.
```suggestion
    def __lt__(self, other: int | float) -> None:
        if isinstance(other, Distance):
            return self.km < other.km
        return self.km < other
```